### ISSUE TITLE
Format lint messages like a C compiler

### ISSUE THREAD

**Issue by rdiez** [stylelint/stylelint#5274]
I am a newcomer and I was surprised at the lint message formatting:

SomeFile1.html
   1:23   ✖  Some warning
   2:34   ✖  Something else

SomeFile2.html
   4:56   ✖  Some other warning

That is hard to parse (for a machine). I guess that most IDEs, like Atom, will have their own plug-in that understands that format. Or maybe there is way to generate that information as a JSON file or whatever.

But wouldn't it be easier to (optionally) generate such messages like a C compiler does?

/home/user/some/file.cpp:123:45: error: some error here

This way, my Emacs would recognise those warnings and automatically hyperlink them.

Emacs is rather flexible about the message formatting, see compilation-error-regexp-alist . But I do not think that this option can adapt to stylelint, because the filename is separate from the warning messages.

**Comment by ybiquitous** [comment]
@rdiez Could you try `-f unix` flag, please?

https://stylelint.io/user-guide/usage/options#formatter

**Comment by rdiez** [comment]
Thanks, I'm sorry I missed that option.
